samples/nrf9160/nrf_cloud_rest_agps/sample.yaml

```diff
@@ -0,0 +1,8 @@
+sample:
+  name: nRF Cloud REST API
+tests:
+  samples.nrf9160.nrf_cloud_rest_agps:
+    build_only: true
+    build_on_all: true
+    platform_allow: nrf9160dk_nrf9160ns
```

Comment:
```suggestion
    platform_allow: nrf9160dk_nrf9160_ns
```

Reply:
Also, it's needed to add `integration_platforms` section to get the sample built by the CI:
```
    integration_platforms:
      - nrf9160dk_nrf9160_ns
```